pages/docs/specifications/v2.0.0.md

```diff
@@ -96,11 +96,11 @@ It combines resource listing and API declaration together into one document.
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="A2SAsyncAPI"></a>asyncapi | [AsyncAPI Version String](#A2SVersionString) | **Required.** Specifies the AsyncAPI Specification version being used. It can be used by tooling Specifications and clients to interpret the version. The structure shall be `major`.`minor`.`patch`, where `patch` versions _must_ be compatible with the existing `major`.`minor` tooling. Typically patch versions will be introduced to address errors in the documentation, and tooling should typically be compatible with the corresponding `major`.`minor` (1.0.*). Patch versions will correspond to patches of this document.
+<a name="A2SAsyncAPI"></a>asyncapi | [AsyncAPI Version String](#A2SVersionString) | <LifeCycle text="required" /> Specifies the AsyncAPI Specification version being used. It can be used by tooling Specifications and clients to interpret the version. The structure shall be `major`.`minor`.`patch`, where `patch` versions _must_ be compatible with the existing `major`.`minor` tooling. Typically patch versions will be introduced to address errors in the documentation, and tooling should typically be compatible with the corresponding `major`.`minor` (1.0.*). Patch versions will correspond to patches of this document.
```

Comment:
If you manage to make it, I'm all ears 😄 The question is: is it worth the effort? :)

Reply:
I dont think so, and it will take good efforts. 😅
- Its just a small class change.
- Not having much value addition.

Reply:
>You can't use MDX here. The spec files are [automatically copied from the spec repo](https://github.com/asyncapi/spec/blob/master/.github/workflows/new-spec-release.yml).

Yupp, I understand that it is not an MDX but we are still using many of the components in `docs` and `blogs` to visualize the components. And there can't be text color coding in MD files manually by adding `div` tag. If the custom CSS classes don't work in this md, we surely have to go for another implementation.

>@fmvilas what you think about running a script post loading of a page, which checks the dom nodes with the text content matching to the life cycle parameters.... And adds span tags with classes around it.

This is BTW Nice idea to implement but I don't think it's reliable. It is because `REQUIRED` and `OPTIONAL` are not only life cycle parameters, they are also normal language words used to define certain parameters in normal English. If we made a script, it will take all the words naming `DEPRECATED`, `REQUIRED` and `OPTIONAL` from the spec file and will make it highlighted. There should be any way to differentiate the lifecycle terms from normal English using words. Secondly, we are only making it highlighted in the docs that usually require only certain classes to be implemented, then writing scripts and managing edge cases for it is worth it in efforts?